tests/behat/features/gridfield-inline-delete.feature

```diff
@@ -16,7 +16,6 @@ Feature: GridField inline delete
     And I go to "/admin/test"
 
   Scenario: I can see toast message when I successfully delete a record by clicking the Delete button in action menu
-  Given I take a screenshot after every step
```

Comment:
This shouldn't be there so cleaning that up while I'm here.

Noticed it because I was copying some of the background from this feature.